sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -590,6 +590,12 @@ def _add_argparse_args(cls, parser):
          'enabled with this flag. Please sync with the owners of the runner '
          'before enabling any experiments.'))
 
+  def add_experiment(self, experiment):
```

Comment:
This is great. Please file an JIRA to convert existing uses of `self.debug_options.experiments.append(...)` across the codebase to use this.